src/main/java/com/flansmod/common/teams/CommandTeams.java

```diff
@@ -146,17 +146,32 @@ public void processCommand(ICommandSender sender, String[] split) {
             return;
         }
         if (split[0].equals("addMap")) {
-            if (split.length < 3) {
-                sender.addChatMessage(new ChatComponentText("You need to specify a map name"));
+            if (split.length < 9) {
```

Comment:
eh would it be possible to keep the old one too, and have them as optional args?

Also, if invalid syntax, print correct usage of the command